src/ray/raylet/scheduling_resources.cc

```diff
@@ -361,7 +477,7 @@ ResourceIdSet ResourceIdSet::Acquire(const ResourceSet &resource_set) {
   return ResourceIdSet(acquired_resources);
 }
 
-void ResourceIdSet::Release(const ResourceIdSet &resource_id_set) {
+void ResourceIdSet::Release(const ResourceIdSet &resource_id_set, bool strict) {
```

Comment:
`strict` -> `allow_new_resources` (which will require flipping the if statement)